src/main/kotlin/objects/packets/objects/BoardState.kt

```diff
@@ -55,4 +55,9 @@ class BoardState {
 
     @Required
     var max_ram = arrayOf(1, 1)
+
```

Comment:
`BoardState` is directly sent to the clients (after a `GetBoardStatePacket`, not used currently but will be in the future) and thus shouldn't contain implementation details of the server.

The server has to loop over all the cards multiple times anyways so this doesn't save that much performance over searching for them each turn.